src/components/Breadcrumbs/BreadcrumbItem.js

```diff
@@ -1,42 +1,88 @@
 import React from "react";
 import PropTypes from "prop-types";
-import typography from "../../theme/typography";
-import { Link, StyledText } from "../Text";
+import { Link } from "../Text";
 
-const Text = StyledText.withComponent("span");
-
-const BreadcrumbItem = ({ position, children, href, ...props }) => {
-  const Tag = href ? Link : Text;
-
-  return (
-    <li
-      itemProp="itemListElement"
-      itemScope
-      itemType="http://schema.org/ListItem"
-      style={{ whiteSpace: "nowrap" }}
+const BreadcrumbItem = ({
+  position,
+  children,
+  href,
+  size,
+  weight,
+  ...props
+}) => (
+  <li
+    itemProp="itemListElement"
+    itemScope
+    itemType="http://schema.org/ListItem"
+    style={{ whiteSpace: "nowrap" }}
+  >
+    <Link
+      {...props}
+      itemProp="item"
+      size={size}
+      weight={weight}
+      variant="light"
+      primary
+      accent={null}
+      href={href || null}
     >
-      <Tag
-        itemProp="item"
-        fontSize={typography.size.uno}
-        href={href || null}
-        {...props}
-      >
-        {children}
-      </Tag>
-      {children && <meta itemProp="name" content={children} />}
-      {position && <meta itemProp="position" content={position} />}
-    </li>
-  );
-};
+      {children}
+    </Link>
+    {children && <meta itemProp="name" content={children} />}
+    {position && <meta itemProp="position" content={position} />}
+  </li>
+);
 
 BreadcrumbItem.propTypes = {
   position: PropTypes.string.isRequired,
   href: PropTypes.string,
-  children: PropTypes.node.isRequired
+  children: PropTypes.node.isRequired,
+  size: PropTypes.oneOf([
+    "mini",
+    "uno",
+    "hecto",
+    "kilo",
+    "giga",
+    "tera",
+    "zetta"
+  ]),
+  responsiveSize: PropTypes.shape({
+    small: PropTypes.oneOf([
+      "mini",
+      "uno",
+      "hecto",
+      "kilo",
+      "giga",
+      "tera",
+      "zetta"
+    ]),
+    medium: PropTypes.oneOf([
+      "mini",
+      "uno",
+      "hecto",
+      "kilo",
+      "giga",
+      "tera",
+      "zetta"
+    ]),
+    large: PropTypes.oneOf([
+      "mini",
+      "uno",
+      "hecto",
+      "kilo",
+      "giga",
+      "tera",
+      "zetta"
+    ])
```

Comment:
We should probably make these text size arrays into constant vars sooner rather than later